src/stores/utils.js

```diff
@@ -44,387 +46,1618 @@ export const getCrowdsaleAssets = async networkID => {
 
 async function getCrowdsaleAsset(contractName, stateProp, networkID) {
   console.log(contractName, stateProp, networkID)
+  // prettier-ignore
   const whenSrc =
-    stateProp === 'MintedCappedProxy' ? setFlatFileContentToState(`./contracts/${stateProp}.sol`) : Promise.resolve()
+    stateProp === 'MintedCappedProxy' || stateProp === 'DutchProxy' ? setFlatFileContentToState(`./contracts/${stateProp}.sol`) : Promise.resolve()
+  // prettier-ignore
   const whenBin =
-    stateProp === 'MintedCappedProxy' ? setFlatFileContentToState(`./contracts/${stateProp}.bin`) : Promise.resolve()
+    stateProp === 'MintedCappedProxy' || stateProp === 'DutchProxy' ? setFlatFileContentToState(`./contracts/${stateProp}.bin`) : Promise.resolve()
   let abi
-  //todo
+  // prettier-ignore
```

Comment:
To fix this errors you can run: `npm run lint -- --fix` after removing the `// prettier-ignore` comments.